torch/csrc/autograd/variable.h

```diff
@@ -295,6 +295,13 @@ struct TORCH_API AutogradMeta : public c10::AutogradMetaInterface {
 ///   + if an in-place op is done on view, in rebase_history() of view, which is
 ///     called after every in-place op in VariableType.cpp, the grad_fn of base
 ///     is updated.
+///   + if a single autograd Node returns multiple differentiable views, if any
+///     output is modified by an inplace operation, the autograd engine will make
+///     an equivalent graph (corresponding to the view operations) without using
```

Comment:
editing suggestion: equivalent graph, where each output is treated as if it were produced by a distinct view operation. This discards the original (e.g., user provided) `grad_fn`